src/phoenix/server/api/routers/v1/prompts.py

```diff
@@ -180,3 +241,17 @@ def _prompt_version_response_body(
             output_schema=output_schema,
         )
     )
+
+
+def _prompt_from_orm_prompt(orm_prompt: models.Prompt) -> Prompt:
+    source_prompt_id = (
+        str(GlobalID(PromptNodeType.__name__, str(orm_prompt.source_prompt_id)))
+        if orm_prompt.source_prompt_id
+        else None
+    )
+    return Prompt(
+        id=str(GlobalID(PromptNodeType.__name__, str(orm_prompt.id))),
+        source_prompt_id=source_prompt_id,
+        name=orm_prompt.name,
+        description=orm_prompt.description,
+    )
```

Comment:
What do we think about attaching the list of prompt version Ids here as well? Without this I have no ability to traverse my prompts via rest api alone

Reply:
we probably need a prompt versions listing, I worry about putting the prompt version ids on this b/c even if it's not an issue for now, we might want to paginate that list as well :(